kibble/models/film.go

```diff
@@ -49,6 +49,7 @@ type Film struct {
 	Subtitles       []string
 	SubtitleTracks  []SubtitleTrack
 	CustomFields    CustomFields
+	Refs            CustomFields
```

Comment:
```suggestion
	Refs            Refs
```

Reply:
As the JSON schema from meta says, refs is a map of a string to a string. CustomFields can be anything currently (interface{}), which is why there are methods for getting values as ints or bools.

There are a couple of solutions here:
1. Just use a `map[string]string` and then anytime you want to grab one you can use `film.Refs["letterboxd_id"]`.
2. Create a proper struct `type Refs struct { LetterboxdID string `json:"letterboxd_id"` } and then wire them up properly in the api methods.

As the json schema in meta has explicit mention of a single key (letterboxd_id) in this map, it kind of makes sense to put the same explicit rules onto this type here too.